### PR TITLE
Fix typos in Interoperability LIPs

### PR DESCRIPTION
**Pull request by ricott1**

### CODE REVIEW

**Review by AndreasKendziorra — APPROVED** [review]

File: proposals/lip-0053.md [hunk 1]
```diff
@@ -717,7 +717,7 @@ def execute(trs: Transaction) -> None:
             # In this case, we do not even update the chain account with the new certificate.
             return
         try:
-            # The CCM must come be directed to the sidechain, unless it was bounced on the mainchain.
+            # The CCM must be directed to the sidechain, unless it was bounced on the mainchain.
```
Comment (gkoumout) [inline proposals/lip-0053.md:720]: this will be over-written with the refactoring PR and may cause merge conflicts.

**Review by gkoumout — APPROVED** [review]
Fine from my side, just left a small comment.

File: proposals/lip-0053.md (hunk 1 above)
Comment (ricott1) [reply proposals/lip-0053.md:720]: This should be fixed by the typo fixing PR

**Review by janhack — APPROVED** [review]
Editorial approval

**Pull request merged**